.github/workflows/include-deploy-release.yml

```diff
@@ -108,5 +108,5 @@ jobs:
          "${{ secrets.DOCS_S3_ACCESS_KEY }}" "${{ secrets.DOCS_S3_SECRET_KEY }}" \
          "${{ secrets.SLACK_URL_RELEASE }}" "${{ secrets.SLACK_URL_CI }}" \
          "${{ secrets.GRADLE_PORTAL_KEY }}" "${{ secrets.GRADLE_PORTAL_SECRET }}" \
-         '-PsignBuild=true -PsignSecretRingFileKotlin="${{ secrets.GPG_SIGNING_KEY_BASE_64_DBG }}" -PsignPasswordKotlin=${{ secrets.GPG_PASS_PHRASE_DBG }}'
+         '-PsignBuild=true -PsignSecretRingFileKotlin="${{ secrets.GPG_SIGNING_KEY_BASE_64_DBG }}" -PsignPasswordKotlin=${{ secrets.GPG_PASS_PHRASE_DBG }} -Prealm.kotlin.copyNativeJvmLibs=linux,windows'
```

Comment:
Shouldn't we also include `macos`? I see that the Macos `dylib`-file was already included in previous release, but I can't really grasp why. Was it unintentionally (or intentionally) rebuilt as part of the release 🤔

Reply:
no, macOS file doesn't need to be copied because it is already built. I checked manually with a staged build on https://oss.sonatype.org/ and all of them are uploaded